config/solr_configs/solrconfig.xml

```diff
@@ -38,22 +38,24 @@
     </autoCommit>
   </updateHandler>
 
-  <updateRequestProcessorChain name="add_hashed_id">
-    <processor class="solr.processor.SignatureUpdateProcessorFactory">
-      <bool name="enabled">true</bool>
-      <str name="signatureField">hashed_id_ssi</str>
-      <bool name="overwriteDupes">false</bool>
-      <str name="fields">layer_slug_s</str>
-      <str name="signatureClass">solr.processor.Lookup3Signature</str>
-    </processor>
+  <updateProcessor class="solr.processor.SignatureUpdateProcessorFactory" name="add_hashed_id">
```

Comment:
I'm good with this approach, but I think we ended up removing the processor factory in prod.. and now I can't remember the reason why we did this. @cbeer do you remember?

Reply:
Yeah, it doesn't play nice with atomic updates.

Reply:
Ah that was it. :(

Reply:
Ah, okay, I was wondering about that.  Does that mean that we can't use any processor factory (e.g. `RemoveBlankFieldUpdateProcessorFactory`?)

Reply:
I suspect it was specific to the `SignatureUpdateProcessorFactory`, but maybe filtering at index time would be better? (or, maybe without an XSLT in the mix, it's no longer an issue?)

Reply:
The content from the other institutions will still be an issue.  We index those records through a rake task provided by GeoBlacklight.  I'm not sure if we can dictate that all consumers of that indexing rake task should not receive the blank values or not, but 🤷‍♂️

Reply:
This solution would handle other organizations metadata too, so if this worked w/ atomic updates I would be inclined to adopt something like this. Unless if when we look at reindexing other orgs, we can remove their empties then?

Reply:
🤷‍♂️ only one way to find out.

Reply:
I'll deploy this branch to stage and re-index the content from external providers and see what happens. 🤞

Reply:
And make a PR to sul-solr-configs?

Reply:
![definitely](https://i.kym-cdn.com/entries/icons/facebook/000/022/138/highresrollsafe.jpg)

Reply:
Re-ran indexing of OGM content in earthworks-stage and things appear to have worked well (assuming the issue isn't that atomic updates come around and blow things away later because of how I've configured the update handler).

## [nyu-2451-43780 (prod)](https://earthworks.stanford.edu/catalog/nyu-2451-43780)
<img width="863" alt="nyu-2451-43780-before" src="https://user-images.githubusercontent.com/96776/86073537-d0120c00-ba38-11ea-9073-0b7f1239b58d.png">

## [nyu-2451-43780 (stage)](https://kurma-earthworks-stage.stanford.edu/catalog/nyu-2451-43780)
<img width="855" alt="nyu-2451-43780-after" src="https://user-images.githubusercontent.com/96776/86073561-dbfdce00-ba38-11ea-8ad1-3554277d31bb.png">